source/util.js

```diff
@@ -8,14 +8,14 @@ const ow = require('ow');
 const pkgDir = require('pkg-dir');
 
 exports.readPkg = (packagePath = pkgDir.sync()) => {
+	if (!packagePath) {
```

Comment:
Could you do this instead, in case the passed options.contents causes failures?
```suggestion
exports.readPkg = (packagePath) => {
	if (!pkgDir.sync(packagePath)) {
```

Reply:
What do you think about 09089c60651325f583577e416641fa2117d45e65? I did things a bit differently in order to keep things the way they are.

Reply:
I like it!